docs/quickstart.rst

```diff
@@ -35,15 +35,17 @@ one field: ``hello`` and an input name. And when we query it, it should return `
 
 .. code:: python
 
-    import graphene
+    from graphene import ObjectType, Schema, String
```

Comment:
This is what I mean about adding imports in docs - we should be doing it consistently.

Reply:
Agreed. See above.

Reply:
Is there a sphinx or other tool for actually embedding live code from, say, `examples/`, for inclusion inline in the docs?

Reply:
This might be what you're looking for @changeling `literalinclude`:

https://quick-sphinx-tutorial.readthedocs.io/en/latest/code.html

This option `emphasize-lines` is interesting too.